src/context/index.ts

```diff
@@ -51,14 +51,18 @@ export class Context {
 
   private currentIdentity?: Identity;
 
-  public harvesterClient: ApolloClient<NormalizedCacheObject>;
+  private _harvesterClient?: ApolloClient<NormalizedCacheObject>;
```

Comment:
If you define this as `private _harvesterClient: ApolloClient<NormalizedCacheObject> | null;` you don't have to wrap the assignment in line 63 in an if clause